.gitignore

```diff
@@ -7,3 +7,4 @@ public/
 static/admin/*.bundle.*
 .DS_Store
 yarn-error.log
+.idea
```

Comment:
usually only project specific folders are added to .gitignore

if you want to ignore webstorm files across all your projects you should create global .gitignore